cpp/daal/include/services/internal/sycl/execution_context_sycl.h

```diff
@@ -195,12 +195,17 @@ class OpenClKernelFactory : public Base, public ClKernelFactoryIface
     ::sycl::queue & _deviceQueue;
 };
 
-class SyclExecutionContextImpl : public Base, public ExecutionContextIface
+class [[deprecated("CPP SYCL interfaces have been deprecated as of 2024.0 release.")]] SyclExecutionContextImpl : public Base,
+                                                                                                                  public ExecutionContextIface
 {
 public:
-    explicit SyclExecutionContextImpl(const ::sycl::queue & deviceQueue)
+    explicit SyclExecutionContextImpl(const ::sycl::queue & deviceQueue, const bool fromPython = false)
```

Comment:
Why we are doing this in runtime? 
This is not required.
You already added deprecation warnings.

Reply:
This approach was approved by managers during sync